spring-boot-starter-camunda-sdk/src/test/java/io/camunda/zeebe/spring/client/config/CredentialsProviderSaasTest.java

```diff
@@ -45,31 +54,52 @@
       "camunda.client.auth.client-secret=my-client-secret",
       "camunda.client.auth.issuer=http://localhost:14682/auth-server"
     })
+@EnableConfigurationProperties({
+  ZeebeClientConfigurationProperties.class,
+  CamundaClientProperties.class
+})
 @WireMockTest(httpPort = 14682)
-public class AuthenticationConfigurationSaasTest {
+public class CredentialsProviderSaasTest {
```

Comment:
💭 What do you think about adding a similar test with legacy properties ? Something like `LegacyCredentialsProviderSaasTest`

Reply:
Done 👍